srcpkgs/python3-MechanicalSoup/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-MechanicalSoup'
+pkgname=python3-MechanicalSoup
+_pkgname=MechanicalSoup
+version=0.11.0
+revision=1
+wrksrc=${_pkgname}-${version}
```

Comment:
just write it out here, there is only one usage of _pkgname, it is not worth declaring a variable just to use it once